starter-code/client/src/app/components/home-page/home-page.component.html

```diff
@@ -0,0 +1,13 @@
+<h1>Journal entries: </h1>
+
+<div class="loading" *ngIf="!entries">
+  loading
+</div>
+
+<ul *ngIf="entries">
+
+  <li *ngFor="let entry of entries">
+   <app-entry-list [entry]="entry"></app-entry-list>
```

Comment:
Since is a single "entry", the component name should be something like `app-entry-card`.